flink/1.8/README.md

```diff
@@ -76,6 +79,33 @@ core@ip-10-0-6-55 ~ $ docker run -it mesosphere/dcos-flink:1.2.0-1.2 /bin/bash
 
 root@2a9c01d3594e:/flink-1.2.0# ./bin/flink run -m <jobmangerhost>:<jobmangerjobmanager.rpc.port> ./examples/batch/WordCount.jar --input file:///etc/resolv.conf --output file:///etc/wordcount_out
 ```
+### Running with HA
```

Comment:
There are a few spelling mistakes in the below draft.

For clarity would you mind stating that the below is the client configuration.